src/store/concepts/movieCatalogs/operations/getTrendingMoviesOperation.js

```diff
@@ -0,0 +1,38 @@
+import { createLogic } from 'redux-logic'
+import { storeData } from 'Store/data/actions'
+import { path } from 'ramda'
+import { normalize, schema } from 'normalizr'
+import { ALL_TRENDING_DAY_URL, SEARCH_MOVIE_URL, TRENDING_MOVIES } from 'Constants'
+import { GET_MOVIES } from '../types'
+import { requestMoviesStart, requestMoviesSuccess, requestMoviesError } from '../actions'
+
+export const getMoviesOperation = createLogic({
+  type: GET_MOVIES,
+  latest: true,
+  async process({ getState, action, axios }, dispatch, done) {
+    let { queries } = action
+    const searchQuery = path(['movieCatalogs', `${TRENDING_MOVIES}`, 'meta', 'search'], getState())
+    const url = searchQuery ? SEARCH_MOVIE_URL : ALL_TRENDING_DAY_URL
+    if (searchQuery) {
+      queries = { ...queries, query: searchQuery }
+    }
```

Comment:
need to be discussed

Reply:
!rename to params

Reply:
`let { queries } = action`
I need `let` here because queries can be `undefined`